scripts/install-aspnet-codegenerator.cmd

```diff
@@ -1,5 +1,5 @@
-set VERSION=9.0.0-dev
-set DEFAULT_NUPKG_PATH=%userprofile%/.nuget/packages
+set VERSION=9.0.1-dev
```

Comment:
Should this be bumped to 9.0.1-dev now if we haven't shipped 9.0.0 yet? Its just for testing I guess so doesn't matter much, just curious the intent.

Reply:
yep left over from testing sorry